slotqueue/slotqueue.go

```diff
@@ -37,7 +37,7 @@ type queue struct {
 // SlotEvent represents the notify event fire for each pubkey subject with the proper duty
 type SlotEvent struct {
 	Slot uint64
-	Duty *ethpb.DutiesResponse_Duty
```

Comment:
@nivBlox I think this whole struct should be inside the operator package or something

Reply:
already changed